Gemfile

```diff
@@ -54,3 +54,5 @@ group :deployment do
   gem 'capistrano-rvm', require: false
   gem 'dlss-capistrano', '~> 3.6', require: false
 end
+
+gem 'action_policy', '~> 0.5.3'
```

Comment:
I think we've been stuffing our gems that are in every env up above the group-specific blocks. Open to moving this up above `bootsnap` or do you think there is good reason to put this down here?

Reply:
This is what happens when you use the CLI: `bundle add action_policy`

Reply:
@jcoyne OK, cool. How would you feel about shifting the other gems from the top to the bottom, then, so future CLI appends build on what we already have? It's not a major concern but I also would like to head off any future rehashes of this if possible.